marshal/message.go

```diff
@@ -45,6 +45,10 @@ const (
 	msgTypeClaimingMessages   msgType = 3
 	msgLengthClaimingMessages int     = msgLengthBase + 1
 	idxCMProposedLastOffset   int     = idxBaseEnd + 1
+
+	msgTypeReleaseGroup   msgType = 4
+	msgLengthReleaseGroup int     = msgLengthBase + 1
```

Comment:
This messages needs a "Good to" time in it as well so that the consumers know when they can exit release mode.